bugbug/repository.py

```diff
@@ -24,8 +25,10 @@
 db.register(COMMITS_DB, 'https://www.dropbox.com/s/mz3afgncx0siijc/commits.json.xz?dl=1', 'v1')
 
 COMPONENTS = {}
+COMP_TOUCHED_PREV = defaultdict(int)
```

Comment:
Since these are not constants, make them lowercase (and move them right after the author_experience dicts).